cmd/sharded-test-server/virtual.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/abiosoft/lineprefix"
+	"github.com/fatih/color"
+
+	"k8s.io/klog/v2"
+
+	"github.com/kcp-dev/kcp/cmd/test-server/helpers"
+	"github.com/kcp-dev/kcp/test/e2e/framework"
+)
+
+func startVirtual(ctx context.Context, index int, logDirPath string) (<-chan error, error) {
+	logger := klog.FromContext(ctx)
+	logger.Info("starting virtual-workspaces standalone server", "index", index)
```

Comment:
nit: not sure we do that elsewhere, but maybe print the command line with all parameters instead?

Reply:
https://github.com/kcp-dev/kcp/issues/1858